tests/virt_control_plane_test.go

```diff
@@ -269,7 +273,7 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 				blackHolePodFunc("del")
 
 				By("ensuring we now have a ready virt-handler daemonset")
-				Eventually(readyFunc, 30*time.Second, time.Second).Should(BeNumerically(">", 0))
+				Eventually(readyFunc, 30*time.Second, time.Second).Should(BeNumerically("==", desiredDeamonsSetCount))
```

Comment:
Will 30s be enough to have `desiredDaemonsSetCount` in ready state?

Reply:
I don't have data to back it up but I feel like it should be enough